scripts/lib/sphinx/addFrontMatter.ts

```diff
@@ -39,11 +39,15 @@ python_api_name: ${result.meta.python_api_name}
 ${markdown}
 `;
     } else if (result.isReleaseNotes) {
+      const versionStr = pkg.hasSeparateReleaseNotes
+        ? ` ${pkg.versionWithoutPatch}`
+        : "";
+      const descriptionSuffix = pkg.hasSeparateReleaseNotes
+        ? `in ${pkg.title}${versionStr}`
+        : `to ${pkg.title}`;
```

Comment:
This is a behavior change for Qiskit, but same for Runtime and Provider.

Reply:
Could we use `to` for both cases?

Reply:
The English reads poorly to say "Changes made to Qiskit 0.45" because it sounds like you're only changing Qiskit 0.45, when in reality you are changing Qiskit 0.45+. It's not wrong, but makes less sense.

(Prepositions are the worst part of languages for me lol)